extension/tsconfig.json

```diff
@@ -1,13 +1,10 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "ES2019",
+    "target": "es2020",
     "outDir": "out",
     "lib": [
-      "ES2019",
-      "dom",
-      "es2018.regexp",
-      "ES2020.String"
```

Comment:
- `dom` includes dom definitions, we're not using that in TS.
- `es2018.regexp` my best guess is that this was added for named groups? It seems to have been present since the start, copy paste most likely?
- `ES2020.String` seems like something needed to use class methods of `String` not currently available in ES2018. This was added by me in 3823ab6bb49179abb689711818d80d533b6a2d32 merged in PR #67. I can't really see why it was added. The usage might have been dropped in the process but the type definition remained. In #67 `es6` and `es2018` was switched to `ES2019`

Some more info https://www.typescriptlang.org/tsconfig#lib

Reply:
So hopefully we won't be missing anything :upside_down_face:

Reply:
One thing I noted is that the size of the .vsix files are around 5mb. Hopefully it's just something locally for me. But should definitely be checked out before merge.

Ran both: 
`vsce package --message "1.23.0" --pre-release --baseContentUrl "https://github.com/jwikman/nab-al-tools/raw/master/extension" "1.23.0"`
and
`vsce package --message "1.24.0" --baseContentUrl "https://github.com/jwikman/nab-al-tools/raw/master/extension" "1.24.0"`

Reply:
deleted `dist` and `out` which gave me 114 files and 1.54MB. Better, but something still seems of.

Reply:
I'll look at it later tonight 👍

Reply:
I just think you need to update your branch from my master. 
It's the `.vscodeignore` that needs to be updated to remove the coverage folders...

Reply:
oh yes, 31 files 1.01mb :+1: